### PR TITLE
Docs: improve backup section

### PR DESCRIPTION
**Pull request by math-GH**
Changes proposed in this pull request:

- added some information about the backup

### CODE REVIEW

**Comment by math-GH** [comment]
motivated by #4278 and took some text and information from it

File: docs/en/admins/05_Backup.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # Backup
 
-This tutorial demonstrates commands for backing up FreshRSS. It assumes that your main FreshRSS directory is `/usr/share/FreshRSS`; If you’ve installed it somewhere else, substitute your path as necessary.
+FreshRSS global settings are in `./data/config.php` and users' settings are in `./data/users/*/config.php`. You can also backup the whole `./data/` directory but exclude the things you do not want.
```
Comment (Frenzie) [inline docs/en/admins/05_Backup.md]:
```suggestion
FreshRSS global settings are in `./data/config.php` and users' settings are in `./data/users/*/config.php`. You can also back up the whole `./data/` directory but exclude the things you do not want.
```

File: docs/en/admins/05_Backup.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 
-## Installation Backup
+If you use extensions, than in each directory in `./extensions` the folder `static` contains the user extensions settings.
```
Comment (Frenzie) [inline docs/en/admins/05_Backup.md]: I think this is what's intended? Not 100% sure without checking though.
```suggestion
If you use extensions, then each directory in `./extensions` the folder `static` contains the user's extension settings.
```

File: docs/en/admins/05_Backup.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+
+## Full-Installation Backup
 
 Do this before an upgrade.
 
-### Creating a Backup
+This following tutorial demonstrates commands for backing up FreshRSS. It assumes that your main FreshRSS directory is `/usr/share/FreshRSS`; If you’ve installed it somewhere else, substitute your path as necessary.
```
Comment (Frenzie) [inline docs/en/admins/05_Backup.md]:
```suggestion
This following tutorial demonstrates commands for backing up FreshRSS. It assumes that your main FreshRSS directory is `/usr/share/FreshRSS`. If you’ve installed it somewhere else, substitute your path as necessary.
```

File: docs/en/admins/05_Backup.md [hunk 4]
```diff
@@ -54,6 +58,10 @@ rm FreshRSS-backup.tgz
 
 You can export your feed list in OPML format either from the web interface, or from the [Command-Line Interface](https://github.com/FreshRSS/FreshRSS/blob/edge/cli/README.md).
 
+The OPML export only exports the standard OPML parameters and does not export things such as desired refresh frequency, custom attributes such as passwords, user agent, XPath Web scraping, etc.
+
+To export all that, use a full back-up with export-to-sqlite, as in following sectiong is described.
```
Comment (Frenzie) [inline docs/en/admins/05_Backup.md]:
```suggestion
To export all that, use a full back-up with export-to-sqlite, as described in the following section.
```

**Review by Frenzie — COMMENTED** [review]
Looks great! Left a few minor nits to fix before merge.

**Pull request merged**